frontend/src/components/browse/BrowseContent.tsx

```diff
@@ -200,7 +196,8 @@ export default function BrowseContent({
       };
       setNonFilterParams(splitQueryObject.nonFilters);
       if (splitQueryObject?.nonFilters?.message) {
-        showFeedbackMessage({
+        // ? is needed due to the context type {showFeedbackMessage?: (..) => void;}
+        showFeedbackMessage?.({
```

Comment:
added a proper typing for showFeedbackMessage. This resulted in erros due to showFeedbackMessage being optional within the context. Maybe one could this fix in the future, in a better way. Not the main focus of this PR.